showcase/src/assets/examples/pretty-print/css.html

```diff
@@ -0,0 +1,10 @@
+<h3>Original Code</h3>
```

Comment:
This text is ok, it should not be translated because it is a code example to be shown in the ExampleViewer 👍